internal/engine/compiler/impl_amd64.go

```diff
@@ -4357,9 +4376,12 @@ func (c *amd64Compiler) compileLoadValueOnStackToRegister(loc *runtimeValueLocat
 		inst = amd64.MOVDQU
 	case runtimeValueTypeV128Hi:
 		panic("BUG: V128Hi must be be loaded to a register along with V128Lo")
-	default:
+	case runtimeValueTypeI32, runtimeValueTypeF32:
```

Comment:
this and the same diff in `compileReleaseRegisterToStack` below are the actual change in this patch